app/pages/clinicianedit/clinicianedit.js

```diff
@@ -60,7 +62,13 @@ export const ClinicianEdit = (props) => {
 
   const selectedClinicianRoles = selectedClinician?.roles;
   const clinicianName = selectedClinician?.name;
-  const adminCount = filter(clinicians, { roles: ['CLINIC_ADMIN'] }).length;
+  const activeAdmins = filter(
+    clinicians,
+    (clinician) =>
+      indexOf(clinician.roles, 'CLINIC_ADMIN') !== -1 &&
+      isUndefined(clinician.inviteId)
+  );
+  const adminCount = filter(activeAdmins, { roles: ['CLINIC_ADMIN'] }).length;
```

Comment:
I don't think this needs to be filtered here, since all `activeAdmins` are guaranteed to have the `CLINIC_ADMIN` role.
```suggestion
  const adminCount = activeAdmins.length;
```

Reply:
Ah yeah, that logic moved up so this can be simplified. Good call 👍🏼